index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>freeCodeCamp Survey Form</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+    <main>
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+        <form action="https://freecodecamp.com/submit-survey-form" id="survey-form">
+            <div>
+            <section>
+                <fieldset class="top">
+                    <label id="name-label">Name: <input type="text" ; placeholder="Enter your name" id="name"
```

Comment:
-For HTML best practices, kindly remove the `; ` syntax in between your attributes as isn't necessary.